vendor.conf

```diff
@@ -41,7 +41,7 @@ github.com/containerd/ttrpc a5bd8ce9e40bc7c065a11c6936f4d032ce6bfa2b
 github.com/syndtr/gocapability d98352740cb2c55f81556b63d4a1ec64c5a319c2
 gotest.tools v2.3.0
 github.com/google/go-cmp v0.2.0
-go.etcd.io/bbolt v1.3.2
+go.etcd.io/bbolt 2eb7227adea1d5cf85f0bc2a82b7059b13c2fa68
```

Comment:
do you know if there's a new release/tag planned for this?

Reply:
No idea on how their timeline for cutting new releases is. The complete bbolt test suite passes.

Reply:
Now I think it's fine!